src/common/components/table-pk-picker/table-pk-picker.component.tsx

```diff
@@ -19,7 +20,8 @@ export const TablePkPicker: React.FC<Props> = props => {
   const { name, options, selectTitle, value, onChange, disabled, isError } =
     props;
 
-  const [selectedPath, setSelectedPath] = React.useState('');
+  const path = value && value.label && generateLabel(options, value.id, []);
```

Comment:
I think this can be written more concesily

```diff
- const path = value && value.label && generateLabel(options, value.id, []);
+ const path = value?.label && generateLabel(options, value.id, []);
```

Reply:
It's fixed